PocketKit/Sources/Sync/RemoteMapping/SlateLineup+remoteMapping.swift

```diff
@@ -44,6 +44,9 @@ extension Recommendation {
 
     func update(from remote: RemoteRecommendation, in space: Space) {
         remoteID = remote.id
+        title = remote.curatedInfo?.title
+        excerpt = remote.curatedInfo?.excerpt
+        imageURL = remote.curatedInfo?.imageSrc.flatMap(URL.init)
```

Comment:
For my own knowledge: I just want to make sure I'm interpreting this right - a Recommendation's title, excerpt, and imageURL will be based on its curated info, otherwise its syndicated article? Since we don't store curated info in Core Data, the `curatedInfo` that's obtained via query is the middleman to these values.

Reply:
Right now it will. And semi correct, if there's no curatedInfo, then we see if if its syndicated and if so we use that, then if its not syndicated we use the Item data. 

This is also a bit of foresight, because when we move to new home next year, all of this data will be on the Recommendation with no need for fall backs.